backend/src/libs/packages/database/generated-schema/0005_add_orders_table.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE IF NOT EXISTS
+  "orders" (
+    "id" serial PRIMARY KEY NOT NULL,
+    "price" integer NOT NULL,
+    "scheduled_time" timestamp NOT NULL,
+    "start_point" varchar NOT NULL,
+    "end_point" varchar NOT NULL,
+    "status" "status" NOT NULL,
```

Comment:
is this a valid syntax?

Reply:
yes, but I forgot to export Enum from tables-schema.ts, and it was reason why `CREATE TYPE "status" AS ENUM('pending', 'confirmed', 'canceled', 'done');` was not generated